### PR TITLE
Apply the ginkgolinter linter

### PR DESCRIPTION
**Pull request by nunnatsa**
Force standards of using ginkgo and gomega.

See additional info in https://github.com/nunnatsa/ginkgolinter

Signed-off-by: Nahshon Unna-Tsameret <nunnatsa@redhat.com>

**Reviewer Checklist**

> Reviewers are supposed to review the PR for every aspect below one by one. To check an item means the PR is either "OK" or "Not Applicable" in terms of that item. All items are supposed to be checked before merging a PR. 

- [ ] PR Message
- [ ] Commit Messages
- [ ] How to test
- [ ] Unit Tests
- [ ] Functional Tests
- [ ] User Documentation
- [ ] Developer Documentation
- [ ] Upgrade Scenario
- [ ] Uninstallation Scenario
- [ ] Backward Compatibility
- [ ] Troubleshooting Friendly
  
**Release note**:

```release-note
None
```

### CODE REVIEW

**Review by tiraboschi — APPROVED** [review]

File: pkg/webhooks/mutator/mutator_test.go [hunk 1]
```diff
@@ -53,15 +53,15 @@ var _ = Describe("webhooks mutator", func() {
...
+		Expect(f(s)).To(Succeed())
 	}
 
 	codecFactory := serializer.NewCodecFactory(s)
 	corev1Codec := codecFactory.LegacyCodec(corev1.SchemeGroupVersion)
 
 	Context("Check mutating webhook for namespace deletion", func() {
 		BeforeEach(func() {
-			Expect(os.Setenv("OPERATOR_NAMESPACE", HcoValidNamespace)).To(BeNil())
+			Expect(os.Setenv("OPERATOR_NAMESPACE", HcoValidNamespace)).To(Succeed())
```
Comment (orenc1) [inline pkg/webhooks/mutator/mutator_test.go:64]: Are you sure this is the most appropriate matcher in this case?
From what I understood, `Succeed()` matcher should be used with a function that returns an error-type as its first or only return value.
https://onsi.github.io/gomega/#succeed

Comment (nunnatsa) [reply pkg/webhooks/mutator/mutator_test.go:64]: Totally agree. This is what the linter looks for, and here is the os.Setenv code from the standard library:

```golang
func Setenv(key, value string) error {
	err := syscall.Setenv(key, value)
	if err != nil {
		return NewSyscallError("setenv", err)
	}
	return nil
}
```

Comment (orenc1) [reply pkg/webhooks/mutator/mutator_test.go:64]: ok, cool 👍

**Pull request merged**